src/coreclr/tools/aot/DependencyGraphViewer/DependencyGraphsForm.cs

```diff
@@ -77,5 +88,43 @@ Graph View
 ";
             MessageBox.Show(helpMessage);
         }
+
+        private void browseButton_Click(object sender, EventArgs e)
+        {
+            openFile();
+        }
+
+        private void openFile()
+        {
+            string argPath = textBox1.Text;
+            if (currentFile is null)
+            {
+                DGMLGraphProcessing dgml = new DGMLGraphProcessing(fileCount);
+                currentFile = dgml;
+                fileCount -= 1;
+                dgml.FindXML(argPath);
+                dgml.Complete += (fileID) =>
+                {
+                    Debug.Assert(fileID == currentFile.FileID);
+                    currentFile = null;
+                };
+            }
+            else
+            {
+                MessageBox.Show("File is already processing. Please Wait.");
+            }
+            textBox1.Text = "";
+        }
+
+        private void ETWButton_Click(object sender, EventArgs e)
+        {
+            if (!(TraceEventSession.IsElevated() ?? false))
+            {
+                MessageBox.Show("To turn on ETW events you need to be Administrator, please close the program and run from an Admin process.");
+            } else
+            {
```

Comment:
```suggestion
            }
            else
            {
```

Reply:
I think by default the format command in Visual Studio is Ctrl + K, Ctrl + D and executing that will automatically reformat everything according to your .editorconfig